js/index.js

```diff
@@ -0,0 +1,89 @@
+var generatedNumber,
+    playerChoice,
+    choicesLeft = 5,
+    gameOver = false,
+    $guess = $('#guessButton'),
+    $guessLeft = $('#numberOfGuessesRemaining'),
+    $guessInput = $('#guess'),
+    $message = $('.message'),
+    $reset = $('#reset');
+    gameOverMessage = 'Game over press reset to play again',
+    guessToHi = 'Your guess was to high',
+    guessToLow = 'Your guess was to low',
+    correctGuessMessage = 'You win the number was ';
+var palyerNumb;
+
+// Genertaes number with an even distrubtion
+function randomNumber() {
+  return Math.floor(Math.random() * (100 - 1 + 1)) + 1;
```

Comment:
@chrislindor lol... sometimes we get so entrenched in the code we miss some funny things... look at the math here:
`(100 - 1 + 1)`